other-docs/guides/upgrading/v14-part1.md

```diff
@@ -0,0 +1,5 @@
+#### Multilingual Module
+
+The Multilingual Module has been removed from Altis v14. This provided no specific functionality. Rather some documented guidance. 
+To add multilingual capability to your Altis project you can continue to use off-the-shelf plugins like MultilingualPress.
```

Comment:
Shall we link to the guide on how to use third-party plugins, as well as MLP itself?